session.go

```diff
@@ -184,9 +190,9 @@ func WithMetadata(meta string) ConnectOption {
 // out of date.
 //
 // For now, don't use this outside of official consumers.
-func WithChildClient(clientType, version string) ConnectOption {
+func WithChildClient(clientType, version string, comments ...string) ConnectOption {
 	return func(cfg *connectConfig) {
-		cfg.ClientInfo = append([]clientInfo{{clientType, version}}, cfg.ClientInfo...)
+		cfg.ClientInfo = append([]clientInfo{{clientType, version, comments}}, cfg.ClientInfo...)
```

Comment:
Do we actually need multiple comment support? We can adjust the parser to accommodate server-side, but I'm not sure we really need more than just the one.

Willing to be convinced though!

Reply:
this is just my cheaty way of getting an `Option<string>` type, but i figured while i was making bad api decisions i might as well give away multiple comments for free

Reply:
Ahh, gotcha. Maybe instead of multiple comments, we use Mozilla's (or maybe everyone's? my browser's at any rate) convention and semicolon-space separate them in a single comment? Even if multiple comments are technically supported by both the RFC/backend, a single comment with multiple values feels more "expected"

Reply:
e.g. `Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0`